client/my-sites/themes/theme-options.js

```diff
@@ -66,10 +91,10 @@ const activate = {
 	action: activateAction,
 	hideForTheme: ( state, themeId, siteId ) => (
 		! getCurrentUser( state ) ||
-		( isJetpackSite( state, siteId ) && isJetpackSiteMultiSite( state, siteId ) ) ||
+		isJetpackSiteMultiSite( state, siteId ) ||
```

Comment:
I don't think we should. It's enough with this selector that will call `isJetpackSite` on its own.
If we add `isJetpackSite` here, it would call `getRawSite` and check for `.jetpack` once, and it will do it again when `isJetpackSiteMultiSite` is called.